ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -131,6 +138,16 @@ def aws_cleanup():
         required=False,
         help="The name of the AWS region to delete the resources from"
     )
+    parser.add_argument(
+        '--prefix',
+        action='append',
+        required=False,
+        help="Additional prefix:hour combo to treat as a special rule. "
```

Comment:
what about to use multi line  string with """ ?

Reply:
Done